_studio/shared/umc/codec/vc1_dec/include/umc_vc1_dec_task_store.h

```diff
@@ -182,17 +182,16 @@ namespace UMC
         };
 
         // external memory management. No need to delete memory
-        void operator delete(void *p) THROWSEXCEPTION
+        void operator delete(void *p)
         {
             //Anyway its incorrect when we trying free null pointer
-            if (!p)
-                throw VC1Exceptions::vc1_exception(VC1Exceptions::mem_allocation_er);
+            VM_ASSERT(p);
```

Comment:
Please, add (void)p; after VM_ASSERT that not to generate new warnings

Reply:
yep.

Reply:
Btw, comment here contradict standard. Deleting null pointer is ok and defined.